provider/resources.go

```diff
@@ -61,7 +63,10 @@ func makeResource(mod string, res string) tokens.Type {
 // Provider returns additional overlaid schema and metadata associated with the provider..
 func Provider() tfbridge.ProviderInfo {
 	// Instantiate the Terraform provider
-	p := shimv2.NewProvider(artifactoryProvider.Provider())
+	p := pfbridge.MuxShimWithPF(context.Background(),
+		shimv2.NewProvider(artifactoryProvider.SdkV2()),
```

Comment:
Would like a confirmation on this block specifically - I followed the [playbook here](https://github.com/pulumi/pulumi-terraform-bridge/blob/master/pf/README.md), and referenced this [draft linode PR ](https://github.com/pulumi/pulumi-terraform-bridge/blob/master/pf/README.md).

[Upstream functions for instantiating the framework provider](https://github.com/pulumi/pulumi-terraform-bridge/blob/master/pf/README.md), and for [the original](https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/artifactory/provider/sdkv2.go#L21)

Reply:
This looks right.